x-pack/plugins/beats_management/common/domain_types.ts

```diff
@@ -57,6 +57,7 @@ export interface ReturnedConfigurationBlock
 
 export interface CMBeat {
   id: string;
+  config_status: 'OK' | 'REQUIRES_UPDATE' | 'WAITING_FOR_SUCCESS' | 'UNKNOWN' | 'ERROR';
```

Comment:
I think only `OK`, `UNKOWN/OFFLINE` and `ERROR` statuses are used for now, can we remove the rest or leave them commented out?

Reply:
We can, I was being a little lazy here